travis/setup_dependencies_common.sh

```diff
@@ -9,6 +9,10 @@ conda config --add channels defaults
 
 shopt -s nocasematch
 
+LATEST_ASTROPY_STABLE=1.1
+LATEST_ASTROPY_LTS=1.0
+LATEST_NUMPY_STABLE=1.11
```

Comment:
No, not yet. Depends whether we want to hack around the numpy issues ("stable" should never pick up 1.5 or 1.8), or can rely on that it will be fixed in conda soon?